core/sql/common/ComViewColUsage.h

```diff
@@ -67,7 +88,9 @@ class ComViewColUsage
 
   void packUsage (NAString &viewColUsageStr)
   {
-      char buf[sizeof(Int64)*2 + sizeof(Int32)*3 + 100];
+      // usage contains 2 int64 and 3 int32, 100 chars is big enough to hold 
+      // the string representation
+      char buf[100];
```

Comment:
I count 47 for the text ("viewUID: ..." etc.), 20 for each of the int64's and 11 for each of the int32s. That adds up to 47 + 40 + 33 = 120, + 1 for the null terminator. Seems like buf needs to be at least 121. Sorry, I should have made this calculation more explicit when I commented earlier.